catalog-spec/examples/summaries-s2.json

```diff
@@ -0,0 +1,180 @@
+          "common_name": "green",
+          "center_wavelength": 5.6,
+          "gsd": 10
+        },
+        {
+          "name": "B4",
+          "common_name": "red",
+          "center_wavelength": 6.645,
+          "gsd": 10
+        },
+        {
+          "name": "B5",
+          "center_wavelength": 7.039,
+          "gsd": 20
+        },
+        {
+          "name": "B6",
+          "center_wavelength": 7.402,
+          "gsd": 20
+        },
+        {
+          "name": "B7",
+          "center_wavelength": 7.825,
+          "gsd": 20
+        },
+        {
+          "name": "B8",
+          "common_name": "nir",
+          "center_wavelength": 8.351,
+          "gsd": 10
+        },
+        {
+          "name": "B8A",
+          "center_wavelength": 8.648,
+          "gsd": 20
+        },
+        {
+          "name": "B9",
+          "center_wavelength": 9.45,
+          "gsd": 60
+        },
+        {
+          "name": "B10",
+          "center_wavelength": 1.3735,
+          "gsd": 60
+        },
+        {
+          "name": "B11",
+          "common_name": "swir16",
+          "center_wavelength": 1.6137,
+          "gsd": 20
+        },
+        {
+          "name": "B12",
+          "common_name": "swir22",
+          "center_wavelength": 2.2024,
+          "gsd": 20
+        }
+      ]
+    ],
+    "cube:dimensions": [
+      {
+        "x": {
+          "type": "spatial",
+          "axis": "x",
+          "extent": [-180,180]
+        },
+        "y": {
+          "type": "spatial",
+          "axis": "y",
+          "extent": [-56,83]
+        },
+        "temporal": {
+          "type": "temporal",
+          "extent": ["2015-06-23T00:00:00Z","2019-07-10T13:44:56Z"]
+        },
+        "spectral": {
+          "type": "bands",
+          "values": ["B1","B9","B10"]
+        }
+      },
+      {
+        "x": {
+          "type": "spatial",
+          "axis": "x",
+          "extent": [-180,180]
+        },
+        "y": {
+          "type": "spatial",
+          "axis": "y",
+          "extent": [-56,83]
+        },
+        "temporal": {
+          "type": "temporal",
+          "extent": ["2015-06-23T00:00:00Z","2019-07-10T13:44:56Z"]
+        },
+        "spectral": {
+          "type": "bands",
+          "values": ["B2","B3","B4","B8"]
+        }
+      },
+      {
+        "x": {
+          "type": "spatial",
+          "axis": "x",
+          "extent": [-180,180]
+        },
+        "y": {
+          "type": "spatial",
+          "axis": "y",
+          "extent": [-56,83]
+        },
+        "temporal": {
+          "type": "temporal",
+          "extent": ["2015-06-23T00:00:00Z","2019-07-10T13:44:56Z"]
+        },
+        "spectral": {
+          "type": "bands",
+          "values": ["B5","B6","B7","B8A","B11","B12"]
+        }
+      }
+    ]
+  },
+  "links": [
+    {
+      "href": "item.json",
+      "rel": "item"
+    },
+    {
+      "href": "summaries-s2.json",
+      "rel": "self"
```

Comment:
I think we should only have examples with self links that are absolute, to be in line with our best practices (or requirement? I forget where we landed).

Reply:
I removed the self link as I don't have a reasonable link to put there. Or should I invent one to have an absolute self link?